python/ray/monitor.py

```diff
@@ -34,7 +34,11 @@
 DRIVER_DEATH_CHANNEL = b"driver_deaths"
 
 # xray heartbeats
-XRAY_HEARTBEAT_CHANNEL = b"6"
+XRAY_HEARTBEAT_CHANNEL = str(
+    ray.gcs_utils.TablePubsub.HEARTBEAT).encode("ascii")
```

Comment:
Why are you applying `str` here? That will give something like `"b'6'"` which is almost certainly not what we want.

Reply:
Can you explain?

Reply:
Oh, I misread the parentheses actually. I think what you have is correct.